Gems/ScriptCanvas/Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -0,0 +1,18 @@
+{
+    "Amazon": {
+        "AssetProcessor": {
+            "Settings": {
+                "RC names": {
+                    "glob": "*.names",
+                    "params": "copy",
+                    "productAssetType": "{6A1A3B00-3DF2-4297-96BB-3BA067A978E6}"
+                },
+                "ScanFolder TranslationAssets": {
+                    "watch": "@ENGINEROOT@/Gems/ScriptCanvas/Assets/TranslationAssets",
+                    "recursive": 1,
+                    "order": 1000
+                }
+            }
```

Comment:
We shouldn't add a ScanFolder like this.

A gem automatically gets the folder of <gem-path>/Assets added to it.
So this rules makes a recursive scan folder inside of a recursive folder which can cause problems.
This should be removed

Reply:
If I remove this, none of the files get picked up and the translation database does not get built. Is there something I am missing to ensure the files are scanned?

Reply:
The files will be scanned as the scan folder would be the `<engine-root>/Gems/ScriptsCanvas/Assets`

Any .names files you have underneath that folder would be relative to that folder.
So if you have  `<engine-root>/Gems/ScriptsCanvas/Assets/TranslationAssets/foo.names`, the relative source path would be `TranslationAssets/foo.names`

Perhaps you might need to perform a full scan on the AP or run the AssetProcessorBatch.